### PR TITLE
Add accessors for StateManager

### PR DESCRIPTION
**Pull request by AndreyTretyak**
Add accessors for StateManager

### CODE REVIEW

File: src/Extensions/Hosting.Services/HostBuilderExtensions.cs [hunk 1]
```diff
@@ -43,10 +44,16 @@ public static class HostBuilderExtensions
 		public static IServiceCollection AddOmexServiceFabricDependencies<TContext>(this IServiceCollection collection)
 			where TContext : ServiceContext
 		{
-			collection.TryAddSingleton<Accessor<TContext>, Accessor<TContext>>();
-			collection.TryAddSingleton<IAccessorSetter<TContext>>(p => p.GetService<Accessor<TContext>>());
-			collection.TryAddSingleton<IAccessor<TContext>>(p => p.GetService<Accessor<TContext>>());
+			bool isStatefullService = typeof(StatefulServiceContext).IsAssignableFrom(typeof(TContext));
```
Comment (vlad-ion) [inline src/Extensions/Hosting.Services/HostBuilderExtensions.cs:47]:
```suggestion
			bool isStatefulService = typeof(StatefulServiceContext).IsAssignableFrom(typeof(TContext));
```

File: src/Extensions/Hosting.Services/HostBuilderExtensions.cs [hunk 2]
```diff
@@ -43,10 +44,16 @@ public static class HostBuilderExtensions
...
+
+			if (isStatefullService)
```
Comment (vlad-ion) [inline src/Extensions/Hosting.Services/HostBuilderExtensions.cs:49]:
```suggestion
			if (isStatefulService)
```

File: src/Extensions/Hosting.Services/HostBuilderExtensions.cs (hunk 1 above)
Comment (AndreyTretyak) [reply src/Extensions/Hosting.Services/HostBuilderExtensions.cs]: Fixed, thank you

File: src/Extensions/Hosting.Services/HostBuilderExtensions.cs (hunk 2 above)
Comment (AndreyTretyak) [reply src/Extensions/Hosting.Services/HostBuilderExtensions.cs]: Fixed, thank you

**Pull request merged**